apps/sap-commerce-cloud/src/components/Dialog/Dialog.styles.ts

```diff
@@ -1,9 +1,9 @@
 import { css } from 'emotion';
-import tokens from '@contentful/forma-36-tokens';
+import tokens from '@contentful/f36-tokens';
 
 export const styles = {
   textInput: css({ width: '250px' }),
-  grid: css({ marginTop: tokens.spacingM, paddingLeft: tokens.spacingL }),
+  grid: css({ margin: `${tokens.spacingM} 0`, paddingLeft: tokens.spacingL }),
```

Comment:
This looks like a style change, was it intentional?

Reply:
Yep. When reviewing the portions of the app we updated, we noticed that there was some spacing built into the card before that isn't there with the new component library. We needed to have this margin apply to the top and bottom now that the element below this input section doesn't have top spacing.